src/server/schemas/search_taxa.yaml

```diff
@@ -35,12 +31,9 @@ properties:
   ranks:
     description: Filter the query to include only these ranks. An empty array is ignored.
     type: array
+    items:
+      type: string
+    examples: [['species', 'strain'], []]
     default: []
     items:
       type: string
-  include_strains:
-    description: |
-      true to include strains in the result, regardless of the ranks field.
-      false to perform no special filtering on strains.
-    type: boolean
-    default: false
```

Comment:
No one is using `search_taxa` correctly. (Pretty sure.) But you can just include this in the `'ranks'` ...?